src/i18n/pt-br/nav.ts

```diff
@@ -1,75 +1,78 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	startHere: 'Comece Aqui',
+	welcomeWorld: 'Bem-vindo, Mundo!',
 	'getting-started': 'Introdução',
-	install: 'Instalação',
-	'editor-setup': 'Configuração do Editor',
-	'upgrade-astro': 'Atualize o Astro',
+	'install-and-setup': 'Instalação e Configuração',
+	'guides/deploy': 'Faça deploy do seu site',
+	'guides/migrate-to-astro': 'Migre para o Astro',
+	'blog-tutorial': 'Tutorial: Construa um Blog',
 
 	coreConcepts: 'Principais Conceitos',
 	'concepts/why-astro': 'Por que Astro?',
 	'concepts/islands': 'Ilhas Astro',
 
-	tutorials: 'Tutoriais',
-	'blog-tutorial': 'Construa um Blog',
-
-	basics: 'Fundamentos',
+	basics: 'Aprenda os Fundamentos',
 	'basics/project-structure': 'Estrutura de Projetos',
 	'basics/astro-components': 'Componentes',
 	'basics/astro-pages': 'Páginas',
 	'basics/layouts': 'Layouts',
 	'basics/astro-syntax': 'Sintaxe do Astro',
 	'basics/rendering-modes': 'Modos de Renderização',
+	'guides/imports': 'Importações',
+	'guides/integrations-guide': 'Integrações',
 
-	'guides/dev-toolbar': 'Barra de Ferramentas de Desenvolvimento',
-
-	examples: 'Receitas',
-	'guides/migrate-to-astro': 'Migre para o Astro',
+	content: 'Gerencie seu Conteúdo',
+	'guides/content': 'Criando Conteúdo',
+	'guides/markdown-content': 'Markdown e MDX',
```

Comment:
The English toolbar doesn't mention MDX
```suggestion
	'guides/markdown-content': 'Markdown',
```

Reply:
Agreed, this was part of the previous translation but I probably missed it when going through the keys. Fixed on https://github.com/withastro/docs/pull/8936/commits/ad43bc5a4697248ceb3457096e3190e4a3b83c7c.